ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/layered/AbstractTransactionsLayer.java

```diff
@@ -87,6 +90,7 @@ public AbstractTransactionsLayer(
     metrics.initSpaceUsed(this::getLayerSpaceUsed, name());
     metrics.initTransactionCount(pendingTransactions::size, name());
     metrics.initUniqueSenderCount(txsBySender::size, name());
+    this.blobCache = new BlobCache();
```

Comment:
It is better to create the blob cache, outside, and then pass it as argument to the layers, because creating it here will result in one cache per layers that extend this class, so now there will be 3 different caches.